staff_graded/locale/settings.py

```diff
@@ -25,7 +25,6 @@
 
 USE_I18N = True
 
-USE_L10N = True
```

Comment:
What happened here ?

Reply:
@awais786 it should not be removed for django32 compatibility, i have fixed and add this again.